### DISCUSSION TITLE
자료 공유 가능하신분 계실까요? ㅠ

### DISCUSSION THREAD

**Discussion by rjc90** [naverz/zepeto-studio-kor#569] [5. Q&A]
내가 지정한 오브젝트 에 캐릭터가 앉기/눕기 같은 애니메이션을 넣고싶은데,,
검색해서 따라해봐도 잘안됩니다 ㅠㅠ

혹시, 만들어진 파일 공유 가능하신분 있으실까요? 뜯어보고싶습니다..!

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 문의 주신 내용이 상호작용으로 제스쳐를 사용하는 기능이라면 아래 답변을 참고해 주세요.

https://github.com/naverz/zepeto-studio-kor/discussions/532#discussioncomment-2448533

**Reply by rjc90** [reply]
감사합니다@@
